src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -730,6 +724,8 @@ public static function createActivityFromMail($mail_id, $object_mode = false)
 		unset($data['bcc']);
 
 		$object['to'] = $data['to'];
+		$object['tag'] = [['type' => 'Mention', 'href' => $object['to'][0], 'name' => 'test']];
```

Comment:
Please remove your test data.

Reply:
It was just a cut and paste - it seemed to be there before :-)